### ISSUE TITLE
Monitoring guidance

### ISSUE THREAD

**Issue by Gordonby** [Azure/AKS-Construction#376]
Add more guidance for users that want to use centralised monitoring.
https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings-policy

![image](https://user-images.githubusercontent.com/17914476/188697358-f0a353bb-541f-44f5-a0a1-44e7baaa87b6.png)

**Comment by liamfoneill** [comment]
I think it would be good to have both options - specify at deploy time and/or policy. Also be nice to set the ingestion limits (if deploying a discrete LAW) especially for Dev/Test.

**Comment by Gordonby** [comment]
@liamfoneill agreed. However if the workspace was in a different subscription then that might cause a few complications with the bicep.
